pages.fr/linux/wg.md

```diff
@@ -3,18 +3,22 @@
 > Gestion de la configuration des interfaces WireGuard.
 > Plus d'informations: <https://www.wireguard.com/quickstart/>.
 
-- Check status of currently active interfaces:
+- Vérifier l'état des interfaces actuellement actives:
 
 `wg`
 
-- Générer une clé privée :
+- Générer une clé privée:
 
 `wg genkey`
 
-- Générer une clé publique à partir d'une clé privée :
+- Générer une clé publique à partir d'une clé privée:
 
 `wg pubkey < {{chemin/vers/clé_privée}} > {{chemin/vers/clé_publique}}`
 
-- Générer une clé publique et privée :
+- Générer une clé publique et privée:
```

Comment:
this in French, the space before the `:` is specific French grammar.

Reply:
Ohh sure :) my bad :(